src/test/java/seedu/address/logic/commands/AddMeetingCommandTest.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.AddMeetingCommand.MESSAGE_ADD_MEETING_SUCCESS;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MEETING_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MEETING_BOB;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Meeting;
+import seedu.address.model.person.Person;
+
+public class AddMeetingCommandTest {
```

Comment:
This test class could have some of its test methods renamed to be more descriptive.